x-pack/plugins/observability/public/pages/overview/old_overview_page.tsx

```diff
@@ -195,6 +195,7 @@ export function OverviewPage({ routeParams }: Props) {
                     rangeFrom={relativeStart}
                     rangeTo={relativeEnd}
                     indexNames={indexNames}
+                    stateStorageKey="xpack.observability.overview.alert.tableState"
```

Comment:
Isn't better to share this hardcoded value in a commonplace then be used here and there https://github.com/elastic/kibana/pull/130538/files#diff-ab4cbe22bf9a1c9bbaf3a3cb32b5aa5b0c8b8e33c34b88ed3c99dbde6fedadb9R333

Reply:
The values are different (unless they are used somewhere else that I missed) so I wouldn't put them in a commonplace until they are duplicated somewhere. The only thing I would suggest is to put that in a constant at the top of the file.

Reply:
@fkanout Thanks for the feedback. That's the whole point of this fix. Let me explain more. Before my fix the localStorage key was shared between overview and alerts page (`xpack.observability.alert.tableState`) and this was causing the alerts table columns bug explained in this [issue](https://github.com/elastic/kibana/issues/126765).

With my current change I introduced two different localStorage keys `xpack.observability.alert.tableState` and `xpack.observability.overview.alert.tableState` for alerts page and overview page accordingly.

@estermv I can put the keys in a constant at the top of the files.